CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [3.5.1](https://github.com/algolia/docsearch/compare/v3.5.0...v3.5.1) (2023-06-15)
```

Comment:
The PR #1916 was not respecting conventional commit so it didn't pick up the entry. We can add it manually.

Reply:
Updated! Sorry about that!